layouts/partials/scripts.html

```diff
@@ -71,8 +71,13 @@
 {{ if .Site.Params.prism_syntax_highlighting -}}
   <script src='{{ "js/prism.js" | relURL }}'></script>
 {{ else -}}
-  {{ $c2cJS := resources.Get "js/click-to-copy.js" | minify |  fingerprint }}
-  <script defer src="{{ $c2cJS.RelPermalink }}" integrity="{{ $c2cJS.Data.Integrity }}" crossorigin="anonymous"></script>
+  {{ $c2cJS := resources.Get "js/click-to-copy.js" -}}
+  {{ if hugo.IsProduction -}}
+    {{ $c2cJS = $c2cJS | minify | fingerprint -}}
+  {{ end -}}
+  <script defer src="{{ $c2cJS.RelPermalink }}" {{ with $c2cJS.Data.Integrity -}}
+    integrity="{{ . }}" {{ end -}}
+    crossorigin="anonymous"></script>
```

Comment:
I think that we should define a partial to encode this. We'll want the same logic for most `<script>` tags. I'll create an issue to follow up on this.

Reply:
- #1263